src/components/LanguageListItemCard/LanguageListItemCard.tsx

```diff
@@ -0,0 +1,81 @@
+import { Card, CardContent, makeStyles, Typography } from '@material-ui/core';
+import clsx from 'clsx';
+import { FC } from 'react';
+import * as React from 'react';
+import { LanguageListItemFragment } from '../../api';
+import { KeyValProp } from '../KeyValProp';
+import { CardActionAreaLink } from '../Routing';
+
+const useStyles = makeStyles(({ spacing }) => {
+  return {
+    root: {
+      width: '100%',
+      maxWidth: '400px',
+    },
+    cardContent: {
+      display: 'flex',
+      justifyContent: 'space-between',
+    },
+    rightContent: {
+      display: 'flex',
+      flexDirection: 'column',
+      justifyContent: 'flex-end',
+    },
+    location: {
+      marginBottom: spacing(2),
+    },
+  };
+});
+
+export type LanguageListItemCardProps = LanguageListItemFragment & {
+  className?: string;
+};
+
+export const LanguageListItemCard: FC<LanguageListItemCardProps> = ({
+  className,
+  name,
+  ethnologueName,
+  rodNumber,
+  ethnologuePopulation,
+}) => {
+  const classes = useStyles();
+
+  return (
+    <Card className={clsx(classes.root, className)}>
+      <CardActionAreaLink to="/some-path">
+        <CardContent className={classes.cardContent}>
+          <div>
+            {/* TODO Is name here the name of the project or language. */}
+            {/* Do I use name or display name */}
+            <Typography variant="h4">{name?.value}</Typography>
```

Comment:
> Is name here the name of the project or language.

The language. This card has nothing to do with projects.

> Do I use name or display name

Both. `name` is the real name of the language. `displayName` is a "fake" name used when the user doesn't have permission to see the real name. Kinda like a nickname/username/gamertag.

So use `name` and fallback to `displayName`.

Reply:
Changed to use name and falls back to displayName.